arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -1954,15 +1951,15 @@
         },
         "ForwardTo": {
           "type": "string",
-          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+          "description": "Path to the recipient to which the message is forwarded."
         },
         "IsReadOnly": {
           "type": "boolean",
           "description": "Value that indicates whether the entity description is read-only."
         },
         "LockDuration": {
           "type": "string",
-          "format": "time-span",
+          "format": "string",
```

Comment:
format should be duration. type string is already present. format tells us that the user will provide us a Duration object which needs to be correctly [de]serialized.